### PR TITLE
added Icon library of svg icons

### PR DESCRIPTION
**Pull request by twishasaraiya**
Please review this PR for Issue #10. I have added Chevron up. I am still trying to figure out how to correctly add rotation for chevron icon.  Since other issues are blocked because of this one I didn't want to delay it further. I hope its okay.

### CODE REVIEW

**Comment by Nshul** [comment]
Can we export all these icons as objects and create an index.js file in this directory which exports all of them. It will make imports into any other component quite easy as we would just have to add import { NameOfIcon } from './Icons'

File: components/Icons/Chevron.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Chevron = props => (
+  <svg viewBox="0 0 448 512" {...props}>
+    <path d="M240.971 130.524l194.343 194.343c9.373 9.373 9.373 24.569 0 33.941l-22.667 22.667c-9.357 9.357-24.522 9.375-33.901.04L224 227.495 69.255 381.516c-9.379 9.335-24.544 9.317-33.901-.04l-22.667-22.667c-9.373-9.373-9.373-24.569 0-33.941L207.03 130.525c9.372-9.373 24.568-9.373 33.941-.001z" />
+  </svg>
+);
+
+export default Chevron;
```
Comment (bogas04) [inline components/Icons/Chevron.js]: Could you please add a `direction` prop that can be one of `up`, `down`, `left`, `right`?

**Comment by bogas04** [comment]
@Nshul that's a neat idea, however do keep in mind that importing from index file would mean your chunk/bundle will include all the icons whatsoever. This might become a problem when we target mobiles & use code splitting. Phones can't run too much JS, no matter how fast they are. So while the idea is neat, it has a downside to it. We can solve that problem by using webpack alias for `./components/Icons/Chevron` as `@icons/Chevron`. This way it will also help us reorganize icons without changing all imports, and even move our library out as a package even.

**Comment by Nshul** [comment]
@bogas04 thats great and it is something new for me, I haven't worked with webpack, I will read up on it.

**Comment by twishasaraiya** [comment]
@bogas04 I have added the direction attribute to chevron.Please check if its correct??Let me know if any changes are required

**Comment by bogas04** [comment]
@twishasaraiya Very nice, however it won't do anything coz you've to implement the direction part using CSS transform, based on direction passed.

**Comment by twishasaraiya** [comment]
Okay!! Should I add `propTypes` for `Chevron` as well??

**Comment by Nshul** [comment]
@twishasaraiya You can follow [this](https://github.com/KhalisFoundation/sttm-web/blob/dev/src/js/components/Icons/Chevron.js), just avoid prop types part.

File: components/Icons/Chevron.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const getStyle = direction =>{
```
Comment (bogas04) [inline components/Icons/Chevron.js]: It seems you've not enabled prettier on your editor. Could you please prettify the files? Let me know if you need help. Thank you so much!

**Comment by bogas04** [comment]
@twishasaraiya please run prettier on your files. You can configure VScode to run it on save :)

Also, please create a section in `pages/index.js` where you import all the Icons for demo purpose.

**Comment by twishasaraiya** [comment]
I have made `Chevron.js` prettier and created Icons `section` also. Please review it. Here is how it looks.
upay

**Pull request merged**